doc/index.rst

```diff
@@ -30,16 +30,33 @@ Contents:
 .. toctree::
    :maxdepth: 2
 
+   first_separator
+
    overview
-   users_guide
+   setup
+   lisa_shell
+   transition_guide
+
+   guides_separator
+
    contributors_guide
+   workflows/index
+
+   api_separator
+
+   target
+   workloads
    kernel_tests
-   automated_testing
-   wlgen
-   internals
-   analysis
-   lisa_tests
+   trace_analysis
+   energy_analysis
+   misc_utilities
+
+..
```

Comment:
@douglas-raillard-arm Do you want me to add placeholders for `bisector` & `exekall`?

Reply:
Could be nice, thanks :)